index.js

```diff
@@ -1,10 +1,29 @@
-import { GraphQLID } from 'graphql';
+import {
+  GraphQLID,
+  GraphQLList,
+  GraphQLString,
+  GraphQLInt,
+  DirectiveLocation,
+  GraphQLDirective,
+} from 'graphql';
 import { SchemaDirectiveVisitor } from 'graphql-tools';
 import { createHash } from 'crypto';
 
 class UniqueIdDirective extends SchemaDirectiveVisitor {
+  static getDirectiveDeclaration(directiveName = 'uid', schema) {
```

Comment:
Looks like `schema` is not used?

Reply:
We have to discuss this together with @czystyl, we eventually need to use schema, because the validator should look only for existing "fields" in actual type or query. My implementation is just initial to kick this off.

Reply:
Maybe you can collaborate with us as well, I'm not sure how to tackle it properly, to be honest.

Reply:
I think we don't need to add default parameter because it comes with name declared in the schema - but need to check.